kratos/solving_strategies/convergencecriterias/residual_criteria.h

```diff
@@ -122,11 +122,11 @@ class ResidualCriteria
 
     ///@} 
     ///@name Operators
-    
     ///@{
 
     /**
-     * Compute relative and absolute error.
+     * @brief Criterias that need to be called after getting the solution
```

Comment:
mmm let's make the constructors explicit (here and in all of the classes you touched)

Reply:
Ok